structuralalignment/superposition/theseus.py

```diff
@@ -1 +1,114 @@
-#here is the theseus code
+import atomium
+import subprocess
+
+# usage: Theseus(["3dk3"])
+class Theseus:
+    def __init__(self, pdbs):
+        self.pdbs = pdbs
+        self.fetched_pdbs = []
+        for pdb in pdbs:
+            self.fetched_pdbs.append(atomium.fetch(pdb))
+        self.align()
+
+    def align(self):
+        self.fetched_pdbs_filename = []
+        """
+        for m in range(len(self.fetched_pdbs)):
+            self.fetched_pdbs[m].model.save(self.pdbs[m])
+        """
+        for m in range(len(self.fetched_pdbs)):
```

Comment:
In Python you can iterate directly over the items, no need to use indices at all!
```suggestion
        for pdb in self.fetched_pdbs:
```

Reply:
Accessing parallel lists in a Pythonic way is done with `zip(...)`:

```suggestion
        for pdb_id, fetched_pdb in zip(self.pdbs, self.fetched_pdbs):
```